### DISCUSSION TITLE
Column witdh in sections mode

### DISCUSSION THREAD

**Discussion by systempal** [home-assistant/frontend#21286] [Dashboard]
Can we get a feature that lets us define and tweak the width of columns in sections mode? It would make things way more flexible and user-friendly.

**Comment by noxhirsch** [comment]
It would be really nice if we could have variables for with, height and column count.

**Reply by maciejmiara** [reply]
That would be great, especially if I resize cards in section, I still cannot resize the section itself, so I can't fit more sections in one row.
Also, where there is only one column, it doesn't take all the space it could, and that would be useful, too.

**Reply by gabriel403** [reply]
"Also, where there is only one column, it doesn't take all the space it could, and that would be useful, too." this tbh

**Comment by boozeman** [comment]
I like to see this too. Sections are good, but it really needs some tweaking of width and colum "padding"  For now it wastes a lot of pixels in width compared to identical page made with Masonry.

**Comment by iotazua** [comment]
I need it !!! Change the width

**Comment by dsypniewski** [comment]
I would love it too, but for me personally it would be enough to add half width sections. I wonder if most of those that want something similar would be satisfied with that.

**Comment by boozeman** [comment]
Sections got some love with 2024.8.0 but there's no possibility to change lateral padding/spacing with colums.

Default Masontry
![image](https://github.com/user-attachments/assets/be07b34b-ca38-4bab-9950-1ce610bd4078)


Sections
![image](https://github.com/user-attachments/assets/fcffc128-c9f9-435f-be37-798caf4eeac3)

I think that there's Still too many wasted pixels
